### ISSUE TITLE
Production-ify Leadership Curriculum

### ISSUE THREAD

**Issue by hannahkane** [MozillaFoundation/plan#757]
#### DARCI
Driver: @hannahkane
A: @chrislarry33 
R: @cassiemc @simonwex (to assign team members, when we get to that point)
C: @chadsansing @zee-moz @kaythaney 
I:

#### Overview
One of the major H1 deliverables for MLN is a Leadership Curriculum. @chadsansing, @zee-moz and the Curriculum working group are spearheading this effort. The Curriculum likely includes several modules, each made up of several activities (similar to Web Literacy curriculum, though these activities are likely to be aimed at the self-directed learner, rather than lesson plans to be facilitated by an educator.)

Once the content is close-ish to finalized, we'll need to production-ify this. This work likely includes:
 - [ ] Branding—see the [Design Brief](https://docs.google.com/document/d/1U6ujOTYD2Bb4b-AvLMzDkX87T6NfIPIdyDxcEonpb5s/edit#). Might include color choices and graphic/illustration work.
 - [ ] Design work, though, ideally the curriculum will fit into the standard layouts we're designing for the CMS project
 - [ ] IA—determining where this curriculum lives
 - [ ] Technical Implementation—deciding whether this is a CMS-appropriate project. Note: one of the requirements is that this content is remixable.

#### Key Deadlines

- Ultimate deadline is end of H1
- Initial content likely to be ready in early April

**Comment by hannahkane** [comment]
Moving to future heartbeat, as per email thread with @kaythaney

**Comment by xmatthewx** [comment]
@zee-moz – let's connect to chat about any design this project might require. Want to schedule a Dumbo meeting?

**Comment by zee-moz** [comment]
@xmatthewx Chad and I are going to work through a design brief early next week, but I am working from Boston all nxt week. Could do Skype later in the week, back the following Monday... If you want to talk sooner, could meet this afternoon in Brooklyn (Dumbo or closer to home) after 3 pm

**Comment by xmatthewx** [comment]
Let's try and find time at the end of this week if possible. I'm traveling next week and then on PTO the following. That leaves just a few weeks before we all head to London. 🕐 🕦 ⏰  

If you and I don't connect, I encourage you to chat with @hannahkane and one of the designers just to outline requirements.

**Comment by kaythaney** [comment]
hey gang,

if schedules allow, would love to join for this thursday or friday.

thanks!

On Mon, Apr 25, 2016 at 11:07 AM, matthew w <notifications@github.com>
wrote:

> Let's try and find time at the end of this week if possible. I'm traveling
> next week and then on PTO the following. That leaves just a few weeks
> before we all head to London. 🕐 🕦 ⏰
>
> If you and I don't connect, I encourage you to chat with @hannahkane
> <https://github.com/hannahkane> and one of the designers just to outline
> requirements.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MozillaFoundation/plan/issues/757#issuecomment-214383742>
>



-- 
@kaythaney <http://twitter.com/kaythaney/>
@MozillaScience <http://twitter.com/mozillascience>

**Comment by xmatthewx** [comment]
I met with @zee-moz @chadsansing @kaythaney today to discuss production needs. I'll try and summarize things I discovered below. Feel free to correct or add to my list. 

 - Status
  - The curriculum is awesome. It is also nearly complete content-wise, pending...
   - A review by mid-May from a set of stakeholders
   - Creation of top-priority videos
 - Next Phase
  - Sharing and testing at London all hands
  - Curating and creating additional videos
  - Posting in an online template (via github? via cms?).
  - Posting of one or two modules _might_ possible before London depending on other production priorities. Nice to have, but not critical.
 - Curriculum home
  - Based on recent IA work, the initial home for curriculum will be a page on the Learning site (with many subpages for content).
   - Pages, links, and design elements will build connections between various hubs and this curriculum. 
 - Design needs
  - Template: Probably good to go after a few color decisions
  - Diagrams: 12-15 explanatory diagrams. Can easily fit within a heartbeat
  - Animations: _Might_ be a need for some animated gifs or video. We should confirm these are necessary and create story boards, then evaluate if we should handle in house or via contractor.
  - Videos: Videos throughout will have a casual low-fi feel. Might use some assets to tie them together such as title screen or props.

From my perspective, assets can be added as created pre- or post-London, following a beta-release before any back-to-school PR work. Timeline is TBD – should be set/confirmed with @hannahkane @kaythaney @chrislarry33

**Comment by hannahkane** [comment]
Awesome! 
Are there specific production tickets to file, or are you suggesting we have a discussion about timeline first?

**Comment by xmatthewx** [comment]
Good question. Some of this can happen concurrently, but a timeline would definitely inform things. 

 - **Diagrams**: I think you can schedule diagram design whenever, based on your other priorities. This work is fairly independent.  
 - **Animations** & **Videos**: This will require a bit more assessment before committing design resources.
 - **Template** & **Github vs CMS**: Picking color will be easy. Navigation around modules will require a bit more work. I'll drop that into an IA/Design issue for some advanced thinking. I think IA, choosing Github vs CMS, and timeline are all interdependent.

**Comment by hannahkane** [comment]
@kaythaney @zee-moz @chadsansing - can you say more about what design support you'd want before London? For example, we'll be planning the next heartbeat later this week. Could we put some of the diagrams in the heartbeat? Are they ready for the design stage?

**Comment by chadsansing** [comment]
I would say the GH diagrams, as well as the illustrations in the Convening module. Maybe also the intro and outro videos? @zee-moz, @kaythaney - does that sound right?

**Comment by zee-moz** [comment]
that sounds right to me-- there are several videos in the intro section, but the animated (or lightly animated) one that comes first will need design love.

**Comment by hannahkane** [comment]
Do the videos need design love or animation love, or both?

**Comment by chadsansing** [comment]
I think the first and maybe last videos - the others are meant to be a kind of cinema verite from the community. Right, @zee-moz? [[flagged!]](http://plzkthx.herokuapp.com)

**Comment by zee-moz** [comment]
@chadsansing is right-- we may add more animated videos eventually but those are the only two that need assets. The others are short vlog-type pieces...

**Comment by hannahkane** [comment]
We'll aim to assign a designer for the GH diagrams and illustrations during the May 20th heartbeat. 

@kaythaney - would it be okay to wait on the video support until we clarify whether the need can best be met in-house or by hiring a contractor?

**Comment by kaythaney** [comment]
let's hold on that for now. @zee-moz may take a stab at a storyboard for the interim. thanks @hannahkane !

**Comment by hannahkane** [comment]
@zee-moz and @sabrinang - did you two meet today? Any updates to share about what the deliverable is for this heartbeat?

**Comment by hannahkane** [comment]
Moving to next heartbeat for which the deliverable is: https://github.com/mozilla/learning.mozilla.org/issues/1875

**Comment by hannahkane** [comment]
@zee-moz @kaythaney @chadsansing - what do you think the next step is on this? should we meet up after London to discuss?

**Comment by chadsansing** [comment]
@hannahkane - that sounds right.

**Comment by hannahkane** [comment]
Taking this out of any milestone until we have a chance to discuss next steps.

**Comment by hannahkane** [comment]
Putting this back in for July 22 heartbeat, but need to clarify the deliverable—likely an animated presentation on "ingredients for open"

**Comment by hannahkane** [comment]
I'm going to close this ticket. We're tracking the video production in another ticket. If there are additional tasks to support the OLC production, we should file those separately.
